src/Stratis.Bitcoin.Features.Miner/PowMining.cs

```diff
@@ -98,6 +104,14 @@ public IAsyncLoop Mine(Script reserveScript)
             return this.mining;
         }
 
+        /// <summary>
+        /// Generates specified amount of blocks in specified maximum amount of tries.
+        /// It is possible that less than the required number of blocks will be mined.
```

Comment:
too much of `specified` :-)

also not amount

->

Generates up to a specified number of blocks with a limited number of attempts.

and I would then move the second sentence into the comment of generate parameter